client/src/document/lazy-bcd-table.tsx

```diff
@@ -55,7 +56,9 @@ function LazyBrowserCompatibilityTableInner({ query }: { query: string }) {
   const { error, data } = useSWR(
     query,
     async (query) => {
-      const response = await fetch(`/bcd/api/v0/current/${query}.json`);
+      const response = await fetch(
+        `${BCD_BASE_URL}/bcd/api/v0/current/${query}.json`
```

Comment:
Also needs to be updated here: https://github.com/mdn/yari/blob/ff39b0ffcd5470a95da30a21d3ebeedc6c7f0ff5/client/src/plus/updates/api.ts#L110

Reply:
Oops, good catch.